web-app/packages/app/src/modules/project/views/ProjectCollaboratorsView.vue

```diff
@@ -8,6 +8,7 @@ SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
   <project-collaborators-view-template
     v-if="isProjectOwner"
     @share="openShareDialog"
+    :allow-share="!instanceStore.globalRolesEnabled"
```

Comment:
how about GLOBAL_READ=false which means that all users are just guests without any explicit access to anything - in that case you want to allow sharing, no?

Reply:
ah, I see it now in that function - and yet I'd simplify it to check of READ flag, WRITE=1 READ=0 does not make much sense.

Reply:
same as above... ?

Reply:
let's keep it as it is, I just do not know why would anyone set WRITE true but READ false